test-common/eos-test-utils.c

```diff
@@ -658,6 +666,46 @@ generate_delta_files (GFile *repo,
   return cmd_result_ensure_ok (&cmd, error);
 }
 
+/**
+ * get_last_ref:
+ * @ref_to_commit: (element-type utf8 uint): a #GHashTable mapping refs to commit numbers.
```

Comment:
This says the key type is `utf8`, but the function returns an `OstreeCollectionRef`.